src/test/java/com/wikia/webdriver/testcases/discussions/PostsList.java

```diff
@@ -17,51 +18,97 @@
   private static final String DESKTOP_RESOLUTION = "1366x768";
   private static final String MOBILE_RESOLUTION = "600x800";
 
-  /** ANONS ON MOBILE SECTION */
+  /**
+   * ANONS ON MOBILE SECTION
+   */
 
   @Test
   @Execute(browserSize = MOBILE_RESOLUTION, asUser = User.ANONYMOUS)
-  public void anonSeePostsListLoadedOnMobile() {
+  public void anonUserOnMobileCanSeePostsList() {
     postsListLoads();
   }
 
   @Test
   @Execute(browserSize = MOBILE_RESOLUTION, asUser = User.ANONYMOUS)
-  public void anonCanSortPostsListOnMobile() {
+  public void anonUserOnMobileCanSortPostsList() {
     userCanSortPostsList();
   }
 
-  /** ANONS ON DESKTOP SECTION */
+  @Test
+  @Execute(browserSize = MOBILE_RESOLUTION, asUser = User.ANONYMOUS)
+  public void anonUserOnMobileCanViewMorePosts() {
+    userCanViewMorePosts();
+  }
+
+  @Test
+  @Execute(browserSize = MOBILE_RESOLUTION, asUser = User.ANONYMOUS)
+  public void anonUserOnMobileCanClickUsername() {
+    clickUsernameLoadsUserPage();
+  }
+
+  /**
+   * ANONS ON DESKTOP SECTION
+   */
 
   @Test
   @Execute(browserSize = DESKTOP_RESOLUTION, asUser = User.ANONYMOUS)
-  public void anonSeePostsListLoadedOnDesktop() {
+  public void anonUserOnDesktopCanSeePostsList() {
     postsListLoads();
   }
 
-  /** LOGGED IN USER ON MOBILE SECTION */
+  @Test
+  @Execute(browserSize = DESKTOP_RESOLUTION, asUser = User.ANONYMOUS)
+  public void anonUserOnDesktopCanViewMorePosts() {
+    userCanViewMorePosts();
+  }
+
+  @Test
+  @Execute(browserSize = DESKTOP_RESOLUTION, asUser = User.ANONYMOUS)
+  public void userCanClickBackToWiki() {
```

Comment:
please use the same naming convention as you used in other methods `anonUser...`